openedx/features/genplus_features/genplus/api/v1/serializers.py

```diff
@@ -86,11 +87,28 @@ class ClassListSerializer(serializers.ModelSerializer):
     current_unit = serializers.SerializerMethodField('get_current_unit')
     lesson = serializers.SerializerMethodField('get_lesson')
 
+    def lastest_unlocked_unit(self, gen_class):
```

Comment:
Is this a typo?

Reply:
Kind of utility function for the other two methods.